### DISCUSSION TITLE
Azure Open AI service for students

### DISCUSSION THREAD

**Discussion by Brucegyx** [microsoft/AI-Chat-App-Hack#20] [Q&A]
Hi I'm a full-time graduate student joining this event. In the Azure Open AI access request form, there are fields asking for company emails, names, phone numbers etc. How should I fill in these to get access? Or are there any other alternatives for students?

**Comment by pamelafox** [comment]
Are you requesting access with an Azure student account, or a normal Microsoft/Azure account? 

Azure student accounts and Azure free accounts currently can't use Azure OpenAI, so you can use an openai.com account instead in that case: https://github.com/Azure-Samples/azure-search-openai-demo/?tab=readme-ov-file#existing-openai-resource
The pricing is similar, it's just a different way of authentication.

Students can also apply to Founder Hub if you have a future business idea:
https://www.microsoft.com/en-us/startups

I've also put together this doc about how you can deploy the RAG chat repo for free:
https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/deploy_lowcost.md

**Reply by Brucegyx** [reply]
Thank you for the answers. Mine is a student account, I’ll try using Open AI’s then.
